output_binary.go

```diff
@@ -8,11 +8,11 @@ import (
 
 // BinaryOutputConfig struct for holding binary output configuration
 type BinaryOutputConfig struct {
-	workers        int
-	Timeout        time.Duration
-	BufferSize     int
-	Debug          bool
-	TrackResponses bool
+	workers        int           `json:"output-binary-workers"`
+	Timeout        time.Duration `json:"output-binary-timeout"`
+	BufferSize     int           `json:"output-tcp-response-buffer"`
```

Comment:
```suggestion
	BufferSize     int           `json:"output-binary-tcp-response-buffer-size"`
```

Reply:
I think the current name is fine, suggested name is too long for flag value.